.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  extends: [
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
```

Comment:
I thought the original plan was to use Airbnb style guide. Is this `plugin:prettier/recommended` generally the same

Reply:
Airbnb is a JS style guide, a lot of the styles don't apply to TypeScript and a lot of TypeScripts best practices is not available on Airbnb. These plugins are the recommended ones for integrating EsLint with TypeScript, React and Prettier as recommended by their authors.

Reply:
Nice. Thanks for the clarification.